mq/mq.go

```diff
@@ -96,9 +96,9 @@ func SetupMQTT(fatal bool) {
 	})
 	opts.SetConnectionLostHandler(func(c mqtt.Client, e error) {
 		slog.Warn("detected broker connection lost", "err", e.Error())
-		c.Disconnect(250)
-		slog.Info("re-initiating MQ connection")
-		SetupMQTT(false)
+		// c.Disconnect(250)
+		// slog.Info("re-initiating MQ connection")
+		// SetupMQTT(false)
```

Comment:
should we uncomment this piece of code?

Reply:
Yes,  added a commit to revert the settings back.